src/main/java/lotto/LottoStatics.java

```diff
@@ -4,44 +4,66 @@
 import java.util.List;
 
 public class LottoStatics {
-    private double sum;
-    private HashMap<Number, Integer> statistics;
-    private int price;
-    private int count;
-
-    public LottoStatics(int totalPrice, int lottoPrice) {
-        this.price = lottoPrice;
-        this.count = totalPrice / lottoPrice;
+
+    private HashMap<Number, Integer> basicStatistics;
```

Comment:
EnumMap 을 사용하면 보너스 필드를 별도로 두지 않아도 될 것 같아요! 

```suggestion
    private EnumMap<LottoPricePolicy, Integer> basicStatistics; 
```